apps/v4/assets/css/main.css

```
@@ -435,3 +435,113 @@
+  --shimmer-spread: --value([length], [percentage]);
+}
+
+@utility shimmer-angle-* {
+  --shimmer-angle: calc(--value(integer) * 1deg);
+}
+
+@media (prefers-reduced-motion: reduce) {
+  .shimmer {
+    animation: none;
+    background-image: none;
+    -webkit-text-fill-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)